### PR TITLE
change link locale

### PR DESCRIPTION
**Pull request by linlinzzo**

### CODE REVIEW

File: files/zh-cn/web/guide/ajax/index.html [hunk 1]
```diff
@@ -84,6 +84,6 @@ <h2 id="例子">例子</h2>
...
+<p><a href="/zh-CN/docs/Web/HTML">HTML</a>, <a href="/zh-CN/docs/Glossary/XHTML">XHTML</a>, <a href="/zh-CN/docs/Web/CSS">CSS</a>, <a href="/zh-CN/docs/Web/API/Document_Object_Model">DOM</a>, <a href="/zh-CN/docs/Web/JavaScript">JavaScript</a>, <a href="/zh-CN/docs/Web/XML">XML</a>, <a href="/en-US/docs/nsIXMLHttpRequest" title="en-US/docs/XMLHttpRequest">XMLHttpRequest</a>, <a href="/zh-CN/docs/Web/XSLT">XSLT</a>, <a href="/en-US/docs/DHTML" title="en-US/docs/DHTML">DHTML</a>, <a href="/zh-CN/docs/Web/JavaScriptSame_origin_policy_for_JavaScript">Same Origin Policy</a></p>
```
Comment (linlinzzo) [inline files/zh-cn/web/guide/ajax/index.html:87]: XMLHttpRequest 和 DHTML 这两个链接 404 了，要删掉吗？ @yin1999

Comment (yin1999) [reply files/zh-cn/web/guide/ajax/index.html:87]: We just translate [mdn/content documents](https://developer.mozilla.org/en-US/docs/Web/Guide/AJAX), so if the links has been removed from en-US, it's ok to remove it from zh-CN.

Comment (linlinzzo) [reply files/zh-cn/web/guide/ajax/index.html:87]: > We just translate [mdn/content documents](https://developer.mozilla.org/en-US/docs/Web/Guide/AJAX), so if the links has been removed from en-US, it's ok to remove it from zh-CN.

下次能与我说汉语吗？“遵从原文”就可以搞定。😅

Comment (awxiaoxian2020) [reply files/zh-cn/web/guide/ajax/index.html:87]: FYI: 我们有的时候使用英语交流，这样可能更方便其他成员来 review. @linlinzzo

**Review by yin1999 — APPROVED** [review]

**Comment by awxiaoxian2020** [comment]
It seems there are some flaws.

**Pull request merged**